src/components/Select/Select.jsx

```diff
@@ -549,6 +549,8 @@ export default class Select extends React.Component {
         ref={selectNode => (this.selectNode = selectNode)}
         data-tip={disabled ? '' : tooltip}
         fullWidth={fullWidth}
+        aria-haspopup="true"
+        aria-expanded={this.state.isOpen}
```

Comment:
Hey @albennett I might be being naiive here with my implentation, but isn't all we need to do is define that the button has a popup, and is / isn't expanded? And since all the select button's are expandable, that should apply to all options, so we should be able to then import this into publish to test things out? If not let me know, happy to make adjustments as necessary 😁

Reply:
Hey @karelm thanks for working on this! I think this change could be an improvement for the select component, so I think this could be an impactful a11y change. 🚀 

I think the issue I was running in with was with the Split Button, which is getting used in campaigns. The split button is built with a button wrapper and the dropdown (select) section being a nested div inside the button. When testing it out with a screen reader, I had trouble selecting the dropdown section because it was nested, as it would always select the button wrapper, which would fire off the onClick. (for example, the View Report in campaigns instead of the select options). There may be a way to select nested options with a screen reader, and I'm just not aware of them yet. :) Let me know if you have a different experience! 

From [this example](https://ej2.syncfusion.com/documentation/split-button/accessibility/), it looks like both buttons are on the same level, which allows for each button to be selected. I'm thinking we need to refactor the split button similarly. Happy to chat with you more about this!